aeron-driver/src/main/c/aeronmd.h

```diff
@@ -637,6 +637,14 @@ uint64_t aeron_driver_context_get_nak_unicast_delay_ns(aeron_driver_context_t *c
 int aeron_driver_context_set_nak_unicast_retry_delay_ratio(aeron_driver_context_t *context, uint64_t value);
 uint64_t aeron_driver_context_get_nak_unicast_retry_delay_ratio(aeron_driver_context_t *context);
 
+/**
+ * Max number of active retransmissions tracked for udp streams with group semantics.
+ */
+#define AERON_MAX_RETRANSMITS_ENV_VAR "AERON_RETRANSMITS_ACTIVE_MAX"
```

Comment:
The name of the variable does not match the value, why? For all other parameters we use the `VALUE_ENV_VAR="VALUE"`.